lisa/notifiers/text_result.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import Path
+from typing import Any, List, Type, cast
+
+from dataclasses_json import dataclass_json
+
+from lisa import notifier, schema
+from lisa.testsuite import TestResultMessage
+from lisa.util import constants, dump_file
+
+
+class TextResult(notifier.Notifier):
+    """
+    Creating log notifier to dump text formatted results for easier
+    view in editing mode. The riginal log is complete but too long to
+    check only the summary
+    """
+    
+
+    @classmethod
+    def type_name(cls) -> str:
+        return constants.NOTIFIER_TEXT_RESULT
+
+    @classmethod
+    def type_schema(cls) -> Type[schema.TypedSchema]:
+        return schema.Notifier
+
+    def _received_message(self, message: TestResultMessage) -> None:
+        if message.is_completed:
+            self.result_file.write(f"{message.status.name:<8} {message.message}")
+
+
+    def _subscribed_message_type(self) -> List[Type[notifier.MessageBase]]:
+        return [TestResultMessage]
+
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        result_path = Path("{constants.RUN_LOCAL_PATH}/lisa-{constants.RUN_ID}-result.txt")
+        if result_path.exists:
+            raise Exception
```

Comment:
1. If it needs to raise an exception, add details messages. So when people see the exception, they know what happens.
2. Raise LisaException. It's easy to know, the exception is raised by LISA code, not underlying.
3. As some practice, it doesn't need to check this, if use `open(result_path, "x")`. It means if this file exists, an error will be raised.